qiita_db/test/test_study.py

```diff
@@ -586,26 +585,20 @@ def test_retrieve_data_types_none(self):
         self.assertEqual(new.data_types, [])
 
     def test_retrieve_raw_data(self):
-        self.assertEqual(self.study.raw_data(), [1, 2, 3, 4])
+        self.assertEqual(self.study.raw_data(), [1])
```

Comment:
I removed them in populate_test_db.sql. Keeping them would require adding 3 more prep templates there, which is a lot of work...